questions.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Ask the Candidates a Question!
+---
+
+# {{ page.title }}
+
+The .NET Foundation's Board of Directors will be working with you and for you, and so we want to make sure you have the ability to ask us any questions you might have about our plans or our approach to the Foundation and our potential seat on the board.
```

Comment:
This is a run on sentence. Consider breaking it up a bit, perhaps like the following:

> The .NET Foundation's Board of Directors will be working with you and for you. We want to make sure you have the ability to ask us any questions you might have about our plans or our approach to the Foundation and our potential seat on the board.

Reply:
Agree with @IEvangelist edit.

Reply:
With those edits, I'd say LGTM

Reply:
@IEvangelist @jongalloway I agree too 👍 made the edit.